test/e2e/resources/notebook_instance_lifecycle_config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: NotebookInstanceLifecycleConfig
+metadata:
+  name: $NOTEBOOK_INSTANCE_LFC_NAME
+spec:
+  notebookInstanceLifecycleConfigName: $NOTEBOOK_INSTANCE_LFC_NAME
+  onStart:
+    - content: cGlwIGluc3RhbGwgUElM
```

Comment:
nit: mention the human readable content in comment and what this is
same for onCreate